setup.cfg

```diff
@@ -47,6 +47,7 @@ testing = mock;python_version<"3.3"
           pytest-xdist; platform_python_implementation!="Jython"
           pytest-localserver
           pypiserver
+          black; python_version>="3.6"
```

Comment:
eh, black has nothing to do with testing, use the fix_lint environment 👍

Reply:
Does black auto-install as a downstream dependency somewhere? I had to install it manually.

For that matter, I had to install *tox* manually.

Reply:
Fixed in e0f7394